rollup.config.js

```diff
@@ -1,33 +1,95 @@
-import babel from "rollup-plugin-babel";
+import babel from "@rollup/plugin-babel";
 import { terser } from "rollup-plugin-terser";
 
 export default [
+  {
+    input: "src/single-spa-react.js",
+    output: [
+      {
+        file: "lib/umd/single-spa-react.js",
+        format: "umd",
+        name: "singleSpaReact",
+        sourcemap: true,
+      },
+      {
+        file: "lib/system/single-spa-react.js",
+        format: "system",
+        sourcemap: true,
+      },
+      {
+        file: "lib/esm/single-spa-react.js",
+        format: "esm",
+        sourcemap: true,
+      },
+      {
+        file: "lib/cjs/single-spa-react.cjs",
+        format: "cjs",
+        sourcemap: true,
+      },
+    ],
+    plugins: [babel({ babelHelpers: "bundled" }), terser()],
+  },
   {
     input: "src/single-spa-react.js",
     output: {
-      file: "lib/single-spa-react.js",
-      format: "umd",
-      name: "singleSpaReact",
+      file: "lib/es2015/single-spa-react.js",
+      format: "esm",
       sourcemap: true,
     },
-    plugins: [babel(), terser()],
+    plugins: [
+      babel({ babelHelpers: "bundled" }),
+      terser({
+        ecma: 6,
```

Comment:
The alternative is to create an `es2015` and `esm` version like single-spa does - https://unpkg.com/browse/single-spa@5.5.3/lib/

More info at https://github.com/single-spa/single-spa/pull/517

Reply:
I am a bit confused, there are already entries for `es2015` and `esm`. It is almost the same as in single-spa except for the banners (because I was taking that from single-spa). Or am I missing something?

Reply:
Ah I got confused. You are right, it's already here.